.meta/sinks/humio_logs.toml.erb

```diff
@@ -54,6 +54,17 @@ examples = ["${HUMIO_TOKEN}", "A94A8FE5CCB19BA61C4C08"]
 required = true
 description = "Your Humio ingestion token."
 
+[sinks.humio_logs.options.type]
```

Comment:
Should this be `sourcetype`?

Reply:
I actually map the field while parsing the config from `type` to `sourcetype` but I neglected that `type` is already a field on all sinks so tit was conflicting. I'll need to come up with a better name for it.

Their Splunk HEC collector calls it `sourcetype` (https://docs.humio.com/integrations/data-shippers/hec/) to match Splunk`s usage, but Humio otherwise refers to that field as `#type` (https://docs.humio.com/foundational-concepts/events/#type). I'm hesitant to call it `sourcetype` as vector's sink is otherwise agnostic to the fact that it is ingesting via Humio's Splunk endpoint.

I'm currently leaning towards `eventtype`. What do you think?

Reply:
Sure, we could also consider `event.type`, where event is a namespace we could bucket all event-level settings under. We could do the same with the `splunk_hec` sink.

Reply:
:+1: I called it `event_type`. I hesitated to add an `event` namespace as this seemed like a bigger change that would require more planning and likely affect other sinks.